### PR TITLE
Changelog 2.4.0

### PR DESCRIPTION
**Pull request by vyaghras**
**Issue number:** N/A

Closes #

**Description of changes:**
- Changelog 2.4.0
- Update version to 2.4.0

**Testing done:**
N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
+# v2.4.0 (2024-09-04)
+
+## OS Changes
+* Add ephemeral-storage commands([#15])
+* Add support for Bootstrap commands ([#62])
+* Update runc ([#123])
```
Comment (ginglis13) [inline CHANGELOG.md]: nit: Update runc to 1.1.14

**Review by ginglis13 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,24 @@
+# v2.4.0 (2024-09-04)
```
Comment (ginglis13) [inline CHANGELOG.md]: Let's update this to 09-05

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
+# v2.4.0 (2024-09-05)
+
+## OS Changes
+* Add ephemeral-storage commands([#15]) - thanks @tzneal
+* Add support for Bootstrap commands ([#62], [#127])
+* Update runc to 1.1.14 ([#123])
+* Update kernel 6.1, 5.10, 5.15 ([#126],[#128])
```
Comment (yeazelm) [inline CHANGELOG.md]:
```suggestion
* Update kernels to 5.10.224, 5.15.165 and 6.1.106 ([#128], [#126])
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,28 @@
+# v2.4.0 (2024-09-05)
+
+## OS Changes
+* Add ephemeral-storage commands([#15]) - thanks @tzneal
```
Comment (ginglis13) [inline CHANGELOG.md]:
```suggestion
* Add ephemeral-storage commands ([#15]) - thanks @tzneal
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+* Add support for Bootstrap commands ([#62], [#127])
```
Comment (ginglis13) [inline CHANGELOG.md]:
```suggestion
* Add support for bootstrap commands ([#62], [#127])
```

**Review by ginglis13 — COMMENTED** [review]
two minor nits otherwise LGTM

**Review by yeazelm — APPROVED** [review]

**Review by ginglis13 — APPROVED** [review]

**Pull request merged**